kubetest2-gke/deployer/commandutils.go

```diff
@@ -17,8 +17,10 @@ limitations under the License.
 package deployer
 
 import (
+	"bytes"
 	"fmt"
 	"os"
+	realexec "os/exec"
```

Comment:
`osexec` would probably be a more appropriate alias here

Reply:
Done.